_build

```diff
@@ -8,6 +8,28 @@ set -x
 # Execute Markdown Linter
 mdl _posts _pages --style _linter/markdown-linter-style.rb
 
+# Skips validation checks that take a long time
+htmlproofer_args_extra=""
+if [ "$1" = "--quick" ]; then
+  htmlproofer_args_extra="--disable-external"
+else
+  htmlproofer_args_extra="--check-external-hash"
+fi
+
+# Development build to prep for HTMLProofer
+BUNDLE_GEMFILE=Gemfile bundle exec jekyll serve -B
+pkill -f jekyll
```

Comment:
Can we do this with `jekyll build` instead?

If we need to do it with `serve` it'd prefer to do

```bash
BUNDLE_GEMFILE=Gemfile bundle exec jekyll serve --detach && pkill --full jekyll
```

I prefer to always use long flag names in shell scripts for readability.

Reply:
We can use `jekyll build` if you'd like to separate the `url` value into the `_config_prod.yml` file and leave the `url` in the main config as blank.  I'm actually thinking maybe that is a better solution after having a fresh look...  Either solution accomplishes the same thing though, I think it's a matter of preference and maybe I'm making it more difficult than it needs to be!  

-[ ] I will do an alternative PR quick using the `jekyll build` option with the separated configs so it might be easier to then decide which you'd prefer.

I updated this PR 0f0e83c with the command you suggested now, however for the `pkill` command `--full` didn't work on my mac and couldn't find that in the [man page for pkill](https://linux.die.net/man/1/pkill)?  Good call on using `--detach` though, that is more readable, agreed.